doc/en/components/pie-chart.md

```diff
@@ -164,6 +211,18 @@ Additionally you can use the `LegendItemTemplate` and `LegendItemBadgeTemplate`
 
 <div class="divider--half"></div>
 
+### How to Show Percentage in Pie Chart Legend
+
+<!-- TODO -->
```

Comment:
With % in the Legend

https://share.goabstract.com/9e5d9807-a47c-47c5-ae31-563f4f346f3d?collectionLayerId=a77dfa99-684b-410f-9601-c0837dd741a2&mode=build